odxtools/tablerow.py

```diff
@@ -157,3 +157,10 @@ def structure(self) -> Optional[BasicStructure]:
     def dop(self) -> Optional[DataObjectProperty]:
         """The data object property object resolved by dop_ref."""
         return self._dop
+
+    def __reduce__(self) -> Tuple[Any, ...]:
+        state = self.__dict__.copy()
+        return (self.__class__,
+                (self.short_name, self.long_name, self.description, self.odx_id, self.oid,
+                 self.table_ref, self.key_raw, self.structure_ref, self.structure_snref,
+                 self.dop_ref, self.dop_snref, self.semantic, self.sdgs), state)
```

Comment:
this seem to be the dataclass fields? if yes, this can written down as
```python
[self.__class__] + [ getattr(self, x.name for x in dataclasses.fields(self) ]
```

another question that I have if this will still work once we switch to python 3.10 and add `kw_only=True` to the `@dataclass` decorators...

Reply:
for python 3.10, we will figure out that bridge once we cross it